config.go

```diff
@@ -55,14 +84,29 @@ func LoadConfig(configFile string) (*menderConfig, error) {
 		return nil, err
 	}
 
+	if confFromFile.Servers != nil {
+		// Overwrite "active" server with first one from the list of servers.
+		confFromFile.ServerURL = confFromFile.Servers[0].ServerURL
+		confFromFile.TenantToken = confFromFile.Servers[0].TenantToken
+		for i := 0; i < len(confFromFile.Servers); i++ {
+			if strings.HasSuffix(confFromFile.ServerURL, "/") {
+				confFromFile.Servers[i].ServerURL =
+					strings.TrimSuffix(confFromFile.Servers[i].ServerURL, "/")
+			}
+		}
+	}
+
 	if strings.HasSuffix(confFromFile.ServerURL, "/") {
+		// Remove trailing '/', which is added back in REST-API strings.
 		confFromFile.ServerURL = strings.TrimSuffix(confFromFile.ServerURL, "/")
```

Comment:
Previous comment also related to this section. This will overwrite the first entry from `Servers`, which can be confusing.